inference-engine/src/mkldnn_plugin/perf_count.h

```diff
@@ -46,4 +46,4 @@ class PerfHelper {
 
 }  // namespace MKLDNNPlugin
 
-#define PERF(_counter) PerfHelper __helper##__counter (_counter->PerfCounter());
+#define PERF(_counter) std::unique_ptr<PerfHelper>(new PerfHelper(_counter->PerfCounter()))
```

Comment:
BTW. Do we really need this macro?

Reply:
I think, it looks better when all code about PerfCount is in `perf_count.h` and user shouldn't know how macro `PERF` works. It just calculates perfcounters. May be it's a matter of taste :blush:

Thank you!

Reply:
But now the user must prepare the std::unique_ptr<PerfHelper> in order to use this functionality. So it doesn't look like an full encapsulation in this macro.